### PR TITLE
Add sliding drawer docs

### PR DESCRIPTION
**Pull request by danrr**
Follow up from #286 

Will rebase once that's merged.

### CODE REVIEW

File: packages/bpk-component-sliding-drawer/stories.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import '~bpk-mixins';
```
Comment (k0nserv) [inline packages/bpk-component-sliding-drawer/stories.scss]: Please add the license header to this file

Comment (danrr) [reply packages/bpk-component-sliding-drawer/stories.scss]: fixed in #286

File: packages/bpk-component-drawer/readme.md [hunk 2]
```diff
@@ -46,13 +46,15 @@ class App extends Component {
...
         <BpkDrawer
+          id="my-drawer"
           isOpen={this.state.isOpen}
           onClose={this.onClose}
           title="Drawer title"
           closeLabel="Close drawer"
           getApplicationElement={() =>
             document.getElementById('application-container')
           }
+          hideTitle
```
Comment (k0nserv) [inline packages/bpk-component-drawer/readme.md]: Why showcase the version with hidden title in the docs?

File: packages/bpk-component-drawer/readme.md [hunk 3]
```diff
@@ -46,13 +46,15 @@ class App extends Component {
           <BpkButton onClick={this.onOpen}>Open drawer</BpkButton>
         </div>
         <BpkDrawer
+          id="my-drawer"
           isOpen={this.state.isOpen}
           onClose={this.onClose}
           title="Drawer title"
```
Comment (k0nserv) [inline packages/bpk-component-drawer/readme.md]: `title` is required unless you specify `hideTitle` right?

Comment (danrr) [reply packages/bpk-component-drawer/readme.md]: It's required all the time. `hideTitle` visually hides the title but it needs to be there for a11y.

File: packages/bpk-component-drawer/readme.md (hunk 2 above)
Comment (danrr) [reply packages/bpk-component-drawer/readme.md]: To show it's there? No better reason than that, I can remove it if you want.

File: packages/bpk-component-drawer/readme.md (hunk 3 above)
Comment (k0nserv) [reply packages/bpk-component-drawer/readme.md]: Neat :)

File: packages/bpk-component-drawer/readme.md (hunk 2 above)
Comment (k0nserv) [reply packages/bpk-component-drawer/readme.md]: I think we should showcase the version with non-hidden title here

Comment (danrr) [reply packages/bpk-component-drawer/readme.md]: Alright, I'll remove the line.

**Pull request merged**